packages/ddp-client/common/livedata_connection.js

```diff
@@ -1621,46 +1620,52 @@ export class Connection {
       this._heartbeat.messageReceived();
     }
 
-    if (msg === null || !msg.msg) {
-      // XXX COMPAT WITH 0.6.6. ignore the old welcome message for back
-      // compat.  Remove this 'if' once the server stops sending welcome
-      // messages (stream_server.js).
-      if (!(msg && msg.server_id))
-        Meteor._debug('discarding invalid livedata message', msg);
-      return;
-    }
+    messages = Array.isArray(messages) ? messages : [messages];
+
+    for (var i = 0; i < messages.length; i++) {
+      var msg = messages[i];
```

Comment:
After this line nothing aside from indenting was changed actually, GitHub just does a bad job at highlighting that.

Reply:
Offtopic advise, append `?w=1` to the review url to get rid of whitespace.

https://github.com/meteor/meteor/pull/10478/files/2870711fbb508d8375f413b3e0bf31c81ed29d8f?w=1

Or, install [`refined github`](https://github.com/sindresorhus/refined-github) and have a <kdb>No Whitespace</kdb> button:

![image](https://user-images.githubusercontent.com/1196524/53868942-6fff8700-3ff7-11e9-8643-3a19f5fbc105.png)